articles/evangelist-and-chameleon.md

```diff
@@ -0,0 +1,61 @@
+
+The big challenge of evangelists is that they usually don't know the alternatives. This means that Andrea sounds more like "I don't want to learn about this tool" than like "I know something better". In the worst case, the functionality she points out as the biggest advantage has been implemented in an improved way in the very alternative she wants to replace.
+
+The advantage of the evangelist is that many migrations need the tenacity of an evangelist. Without evangelists, we would still use the tools from the last century.
+
+The disadvantage is that Andrea might overlook a far more important issue or a precondition for the migration.
+
+# Possibility 3: The Tightrope Walker
+
+Where possible, Andrea uses her preferred tool. Whether she adheres to the team's conventions or not is a balancing act:
+
+ * If your environment differs too much from everybody else's, pair programming will become more difficult.
+ * Most companies have rules about what can be installed/used. Because Andrea most likely doesn't know the reasoning for each of these rules, she has to consider carefully which rules she breaks.
+ * A tool you know well and that is configured to fit your preferences can increase your productivity and reduce your frustration.
+
+The difference between a tightrope walker and an evangelist is that the tightrope walker focuses on her own productivity and comfort. She doesn't attempt to convince anybody, but simply uses whatever she prefers (as long as she believes that it is ok within the context).
+
+The advantage of the tightrope walker is that because the specific tools are less important, Andrea can look at the big picture and focus on the most important issues.
+
+The disadvantage is that Andrea might miss an opportunity to improve the situation of the whole team.
+
+# Possibility 4: The Chameleon
+
+Even though Andrea has her preferred tools, she can work with many other tools. She adapts her way of working to fit the tool in use and continues to learn about each tool she uses. Given her experience with various tools, she can choose the tool that fits best to the problem at hand, regardless of her personal preferences.
+
+Like the whiner, the chameleon does not attempt to change anything. But the motivation is completely different: For the chameleon, the choice of the tool far less important than many other things. The chameleon neither attempts to convince anybody nor takes any action to change the situation. This is not due to resignation, but because the chameleon adapts and is happy with many tools.
+
+The advantages and disadvantages of the chameleon are the same as for the tightrope walker with the added advantage that the chameleon has more experience with a broad range of tools.
+
+# Conclusion
+
+Each possibility has strengths and weaknesses that may or may not fit your situation. So choose wisely: What's your intended impact? And which possibility matches your impact best?
+
```

Comment:
I miss the bit about "Knowledge Sharing about tools".  
How do you share knowledge in a team about tools?

Reply:
I ignored this comment because ... that's a topic that would be worth its own article. And I believe that it is mostly unrelated to the possibilities described in the article, good knowledge sharing is necessary with all possibilities.